src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -180,14 +180,18 @@ ServerHandlerImp::onAccept (Session& session,
 
 auto
 ServerHandlerImp::onHandoff (Session& session,
-    std::unique_ptr <beast::asio::ssl_bundle>&& bundle,
+    std::unique_ptr<beast::asio::ssl_bundle>&& bundle,
         http_request_type&& request,
             boost::asio::ip::tcp::endpoint remote_address) ->
     Handoff
 {
+    bool const is_ws =
```

Comment:
Consider adding an `is_ws` bool to the `ws`/`ws2` version of `ServerHandlerImp::onHandoff` (directly below) for consistency.